Tools/sitl_run.sh

```diff
@@ -214,6 +217,12 @@ elif [ "$program" == "jsbsim" ] && [ -z "$no_sim" ]; then
 	fi
 	"${build_path}/build_jsbsim_bridge/jsbsim_bridge" ${model} -s "${src_path}/Tools/jsbsim_bridge/scene/${world}.xml" 2> /dev/null &
 	JSBSIM_PID=$!
+elif [ "$program" == "sih_sim" ] && [ ! -n "$no_sim" ]; then
+	export SIM_MODE="sih_sim"
+	if [ "$model" != "airplane" ] && [ "$model" != "quadx" ]; then
+		echo "Model ${model} not compatible with with sih. sih supports [quadx,airplane]."
```

Comment:
The sih supports a tailsitter now, merged [here](https://github.com/PX4/PX4-Autopilot/pull/18987)

Reply:
Ok, added the option